lib/resources/df.rb

```diff
@@ -0,0 +1,28 @@
+# encoding: utf-8
+# author: Jonathan Morley
+
+module Inspec::Resources
+  class DfResource < Inspec.resource(1)
+    name 'df'
+    desc 'Use the df InSpec resource to test file system disk space usage'
+    example "
+      describe df('/') do
+        its('size') { should be >= 32000 }
+      end
+    "
+    attr_reader :partition
+
+    def initialize(partition)
+      @partition = partition
+      @cmd = inspec.command("df #{@partition} --output=size | sed \"/blocks/d; s/ *//g\"")
```

Comment:
Since you made `partition` an `attr_reader` (and it's available as `partition` as a method argument), you can drop the `@` here.